lms/djangoapps/verify_student/admin.py

```diff
@@ -12,6 +12,14 @@
 )
 
 
+class VerificationDeadlineAdmin(admin.ModelAdmin):
```

Comment:
This needs to be registered (see bottom of file); otherwise, it does not appear.